avaframe/com1DFA/com1DFA.py

```diff
@@ -782,7 +782,7 @@ def placeParticles(massCell, indx, indy, csz, massPerPart, rng, initPartDistType
         proba = nFloat - nPart
         if rng.random(1) < proba:
             nPart = nPart + 1
-        #nPart = (nFloor + rng.binomial(1, proba)).astype('int')
+        # nPart = (nFloor + rng.binomial(1, proba)).astype('int')
```

Comment:
Is this comment still needed, i.e. attacht to the TODO?

Reply:
removed this line and also the TODO is already resolved